docs/application/dotnet/guides/nui/texteditor.md

```diff
@@ -0,0 +1,113 @@
+
+The following table lists the available `TextEditor` properties.
+
+**Table: TextEditor properties**
+
+| Property                           | Type        | Description                              |
+| ---------------------------------- | ----------- | ---------------------------------------- |
+| `Text`                             | String      | The text to display in UTF-8 format.      |
+| `TextColor`                        | Vector4     | The text color.                           |
+| `FontFamily`                       | String      | The requested font family.                |
+| `FontStyle`                        | PropertyMap | The requested font style.                 |
+| `PointSize`                        | Float       | The size of font in points.               |
+| `HorizontalAlignment`              | String      | The line horizontal alignment.            |
+| `ScrollThreshold`                  | Float       | Horizontal scrolling will occur if the cursor is this close to the control border. |
+| `ScrollSpeed`                      | Float       | The scroll speed in pixels per second.    |
+| `PrimaryCursorColor`               | Vector4     | The color to apply to the primary cursor. |
+| `SecondaryCursorColor`             | Vector4     | The color to apply to the secondary cursor. |
+| `EnableCursorBlink`                | Boolean     | Whether the cursor should blink or not.   |
+| `CursorBlinkInterval`              | Float       | The time interval in seconds between cursor on/off states. |
+| `CursorBlinkDuration`              | Float       | The cursor will stop blinking after this number of seconds. |
+| `CursorWidth`                      | Integer     | The cursor width.                         |
+| `GrabHandleImage`                  | String      | The image to display for the grab handle. |
+| `GrabHandlePressedImage`           | String      | The image to display when the grab handle is pressed. |
+| `SelectionHandleImageLeft`         | PropertyMap | The image to display for the left selection handle. |
+| `SelectionHandleImageRight`        | PropertyMap | The image to display for the right selection handle. |
+| `SelectionHandlePressedImageLeft`  | PropertyMap | The image to display for the left selection handle marker. |
+| `SelectionHandlePressedImageRight` | PropertyMap | The image to display for the right selection handle marker. |
+| `SelectionHandleMarkerImageLeft`   | PropertyMap | The image to display for the left selection handle marker. |
+| `SelectionHandleMarkerImageRight`  | PropertyMap | The image to display for the right selection handle marker. |
+| `SelectionHighlightColor`          | Vector4     | The color of the selection highlight.     |
+| `DecorationBoundingBox`            | Rectangle   | The decorations (handles etc) will positioned within this area on-screen. |
+| `EnableMarkup`                     | Boolean     | Whether the mark-up processing is enabled. |
```

Comment:
Above, this sentence is rephrased like this : 

Enable the Markup string, to process text within the Markup tags using DALi application.<br>**Note**: By default, the Markup string is disabled

I think it would be better to choose one sentence and apply this fix to texteditor.md and textfield.md file both. For now, it looks only texteditor.md has rephrased sentence.